src/main/java/hockeyapp/HockeyappApplication.java

```diff
@@ -146,25 +150,31 @@ public String getDisplayName() {
 
         @SuppressWarnings("unused")
         public FormValidation doCheckApiToken(@QueryParameter String value) throws IOException, ServletException {
-            if(value.isEmpty()) {
-                HockeyappRecorder.DescriptorImpl hockeyappRecorderDescriptor =
-                        (HockeyappRecorder.DescriptorImpl) Jenkins.getInstance().getDescriptorOrDie(
-                                HockeyappRecorder.class);
-                String defaultToken = hockeyappRecorderDescriptor.getDefaultToken();
-                if (defaultToken != null && defaultToken.length() > 0) {
-                    return FormValidation.warning("Default API Token is used.");
-                } else {
-                    return FormValidation.errorWithMarkup(
-                            "You must enter an <a href=\"https://rink.hockeyapp.net/manage/auth_tokens\">API Token</a>.");
+            if (value.isEmpty()) {
+                final Jenkins instance = Jenkins.getInstance();
```

Comment:
in 1.625.3 you can use `getActiveInstance()` IIRC. Jenkins instance cannot be really null here